lms/templates/dashboard/certificate_info.underscore

```diff
@@ -0,0 +1,111 @@
+                        <li class="action">
+                            <a class="btn" href="<%- model.cert_status.download_url %>"
+                               title="<%- gettext('This link will open/download a PDF document') %>">
+                                <%= _.sprintf(gettext("Download Your %(cert_name_short)s (PDF)"), {cert_name_short:settings.cert_name_short}) %></a>
+                        </li>
+                    <% } else if (model.cert_status.show_download_url && $.inArray(model.course_mode, ["verified", "professional"])) { %>
+                        <li class="action">
+                            <a class="btn" href="<%- model.cert_status.download_url %>"
+                               title="<%= _.sprintf(gettext('This link will open/download a PDF document of your verified %(cert_name_long)s.'), {cert_name_long:settings.cert_name_long}) %>">
+                                <%= _.sprintf(gettext("Download Your ID Verified %(cert_name_short)s (PDF)"), {cert_name_short:settings.cert_name_short}) %></a>
+                        </li>
+                    <% }
+
+                    if (model.cert_status.show_survey_button) { %>
+                        <li class="action"><a class="cta" href="<%- model.cert_status.survey_url %>">
+                            <%- gettext("Complete our course feedback survey") %></a></li>
+                    <% } %>
+                </ul>
+            </div>
+
+            <% if (model.cert_status.show_download_url && model.cert_status.linked_in_url){ %>
+                <ul class="actions actions-secondary">
+                    <li class="action action-share">
+                        <a class="action-linkedin-profile" target="_blank" href="<%- model.cert_status.linked_in_url %>"
+                           title="<%- gettext('Add Certificate to LinkedIn Profile') %>"
+                           data-course-id="<%- model.course_key %>"
+                           data-certificate-mode="<%- model.cert_status.mode %>"
+                        >
+                            <img class="action-linkedin-profile-img"
+                                 src="images/linkedin_add_to_profile.png"
+                                 alt="<%- gettext('Share on LinkedIn') %>">
+                        </a>
+                    </li>
+                </ul>
+            <% } %>
+
+            <% if (model.cert_status.show_download_url && model.course_mode === 'verified' && model.cert_status.mode === 'honor'){ %>
+                <div class="certificate-explanation">
+                    <%= _.sprintf(gettext('Since we did not have a valid set of verification photos from you when your %(cert_name_long)s was generated, we could not grant you a verified %(cert_name_short)s. An honor code %(cert_name_short)s has been granted instead.'), {cert_name_short:settings.cert_name_short, cert_name_long:settings.cert_name_long}) %>
```

Comment:
This is difficult to read, please update the format to be match line 29 of this file.